src/olympia/versions/views.py

```diff
@@ -99,7 +99,9 @@ def is_appropriate_reviewer(addon, channel):
         use_cdn = False
         has_permission = is_appropriate_reviewer(
             addon, channel
-        ) or acl.check_addon_ownership(request, addon, dev=True, ignore_disabled=True)
+        ) or acl.check_addon_ownership(
+            request, addon, allow_developer=True, allow_mozilla_disabled_addon=True
```

Comment:
do we want to allow_site_permission here too?

Reply:
Yes, we do - I'll change it (They will be pure unlisted add-ons, we'll want developers to be able to download them).